src/ArborX_DistributedTree.hpp

```diff
@@ -153,14 +153,25 @@ DistributedTree<MemorySpace, Enable>::DistributedTree(
                          "ArborX::DistributedTree::DistributedTree::"
                          "rank_bounding_boxes"),
       comm_size);
-  // FIXME when we move to MPI with CUDA-aware support, we will not need to
-  // copy from the device to the host
+#ifdef ARBORX_USE_CUDA_AWARE_MPI
+  Box bottom_tree_bounds;
+  if (!_bottom_tree.empty())
+    Details::expand(bottom_tree_bounds, _bottom_tree.bounds());
+  Kokkos::deep_copy(Kokkos::subview(boxes, comm_rank), bottom_tree_bounds);
```

Comment:
```suggestion
  Kokkos::deep_copy(space, Kokkos::subview(boxes, comm_rank),
                    _bottom_tree.bounds());
  space.fence();
```

Reply:
I am OK with this change, but do we promise that if a hierarchy is empty, its bounding box is invalid? It is in implementation, but I don't think semantically we guarantee that.

Reply:
We guarantee it does not intersect with anything, expanding with another box is equivalent to assigning that other box, etc.
I haven't put more thoughts whether there is good reason not to say empty data structure bounds should be default constructed bounding volume.

Reply:
Appparently, we needed this check (corresponding to https://github.com/arborx/ArborX/pull/174/commits/28cffae5b61cba123b35cdd17de7f5bcbefefa52) to resolve https://cloud.cees.ornl.gov/jenkins-ci/blue/organizations/jenkins/ArborX/detail/PR-174/6/pipeline. If we don't need it in this pull request to make CI happy, I'm fine with the changes @dalg24 is proposing here, too.

Reply:
We would need the check if and only if we are attempting to build an unmanaged view on the root node bounding volume since the empty BVH does not have any node.